api/SVGAltGlyphItemElement.json

```
@@ -24,10 +26,12 @@
             "version_added": false
           },
           "opera": {
-            "version_added": null
+            "version_added": "≤15",
```

```suggestion
            "version_added": "15",
```


Per testing of http://mdn-bcd-collector.appspot.com/tests/api/SVGAltGlyphItemElement in Opera 12.16.